emacs/tern.el

```diff
@@ -99,12 +99,17 @@
   (let* ((script-file (or load-file-name
                           (and (boundp 'bytecomp-filename) bytecomp-filename)
                           buffer-file-name))
-         (bin-file (expand-file-name "../bin/tern" (file-name-directory (file-truename script-file)))))
-    (if (file-exists-p bin-file)
+         (bin-file (and script-file (expand-file-name "../bin/tern" (file-name-directory (file-truename script-file))))))
+    (unless (or (and bin-file (file-executable-p bin-file))
+		(not (executable-find "npm")))
+      (setq bin-file (or (executable-find "tern")
+			 (expand-file-name "tern/bin/tern"
+					   (file-name-directory (shell-command-to-string "npm -g ls --parseable tern"))))))
```

Comment:
This just returns `/home/marijn/prog/node/lib` on my system (and tern isn't even installed there). What is it intended to do?